packages/rest-hooks/src/index.ts

```diff
@@ -32,14 +26,19 @@ import useSelectionUnstable from './react-integration/hooks/useSelection';
 import hasUsableData from './react-integration/hooks/hasUsableData';
 import { StateContext, DispatchContext } from './react-integration/context';
 
+export {
+  Entity as NestedEntity,
```

Comment:
remapping so there's no actual breaking changes for 'rest-hooks' itself (for now)